terraform/modules/hub/analytics.tf

```diff
@@ -0,0 +1,97 @@
+# analytics
+#
+# frontend applications are run on the ingress asg
+
+resource "aws_security_group" "analytics_task" {
+  name        = "${var.deployment}-analytics-task"
+  description = "${var.deployment}-analytics-task"
+
+  vpc_id = "${aws_vpc.hub.id}"
+}
+
+locals {
+  analytics_location_blocks = <<-LOCATIONS
+
+  set $analytics "https://analytics.tools.signin.service.gov.uk";
+  location /analytics {
+    proxy_pass $analytics/matomo.php?$args;
+  }
+
+  location /healthcheck {
+    return 200;
+  }
+
+  LOCATIONS
+
+  nginx_analytics_location_blocks_base64 = "${base64encode(local.analytics_location_blocks)}"
+}
+
+data "template_file" "analytics_task_def" {
+  template = "${file("${path.module}/files/tasks/analytics.json")}"
+
+  vars {
+    deployment             = "${var.deployment}"
+    nginx_image_and_tag    = "${local.tools_account_ecr_url_prefix}-verify-nginx-tls:latest"
```

Comment:
Can we use `nginx_image_identifier` instead of latest

i.e. port the changes made in https://github.com/alphagov/verify-infrastructure/pull/120/files#diff-aa1cac936beae7def197b7d81e072e7dR4 to this PR?

Reply:
You shouldn't have to add another variable, just use the one introduced in the aforementioned pull request